elements/pfelement/src/pfelement.js

```diff
@@ -243,6 +243,24 @@ class PFElement extends HTMLElement {
     }
   }
 
+  /**
+   * This looks in the shadowdom and lightdom to see if it should return the
+   * user supplied template or the default template located in the shadowdom
+   */
+  getTemplate(selector) {
```

Comment:
Since we're not using this yet, can you store this in another branch?  I don't want to mix it in since pfe-clipboard didn't end up needing it after all.